numpyro/compat/infer.py

```diff
@@ -95,3 +97,56 @@ def get_samples(self, num_samples=None, group_by_chain=False):
 
     def summary(self, prob=0.9):
         self._mcmc.print_summary()
+
+
+class SVI(svi.SVI):
+    def __init__(self,
+                 model,
+                 guide,
+                 optim,
+                 loss,
+                 loss_and_grads=None,
+                 num_samples=10,
+                 num_steps=0,
+                 **kwargs):
+        super(SVI, self).__init__(model=model,
+                                  guide=guide,
+                                  optim=optim,
+                                  loss=loss)
+        self.num_steps = num_steps
+        self.svi_state = None
+
+    def evaluate_loss(self, *args, **kwargs):
+        return self.evaluate(self.svi_state, *args, **kwargs)
+
+    def step(self, *args, rng=None, **kwargs):
+        if self.svi_state is None:
+            if rng is None:
+                rng = numpyro.sample('svi.init', dist.PRNGIdentity())
+            self.svi_state = self.init(rng, *args, **kwargs)
+        try:
+            self.svi_state, loss = jit(self.update)(self.svi_state, *args, **kwargs)
+        except TypeError as e:
+            if 'not a valid JAX type' in str(e):
+                raise TypeError('NumPyro backend requires args, kwargs to be arrays or tuples, '
+                                'dicts of arrays.')
+            else:
+                raise e
+        return loss
+
+    def get_param_store(self):
```

Comment:
Can we name this `get_params` instead, so as not to mislead users that there is really a param store?

Reply:
Oppss... I have no idea why I thought that this is going to override `get_params` method of numpyro svi. Python can do it with `super(...)`... I think I need to refresh my mind. :))